pom.xml

```diff
@@ -54,6 +55,48 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Can't see the <skip> flag. Please add for avoiding jooq generation during all build processes